docs/src/use/command-line-interface.md

```diff
@@ -703,6 +704,18 @@ This option causes ESLint to exit with exit code 2 if one or more fatal parsing
 npx eslint --exit-on-fatal-error file.js
 ```
 
+#### `--no-warn-ignored`
+
+**Flat Config Mode Only.** This option suppresses `File ignored by default / File ignored because of a matching ignore pattern` warnings when an ignored filename is passed explicitly. It is useful when paired with `--max-warnings 0` as it will prevent exit code 1 due to the aforementioned warning.
```

Comment:
Just to clean up a bit.
```suggestion
**Flat Config Mode Only.** This option suppresses both `File ignored by default` and `File ignored because of a matching ignore pattern` warnings when an ignored filename is passed explicitly. It is useful when paired with `--max-warnings 0` as it will prevent exit code 1 due to the aforementioned warning.
```